Makefile

```diff
@@ -44,3 +51,11 @@ test:
 	@find * -name '*_test.go' |\
 	sed -e 's@^@github.com/Cloud-Foundations/cloud-gate/@' -e 's@/[^/]*$$@@' |\
 	sort -u | xargs go test
+
+
+dockerpackagebuild:
+	@echo GIT_COMMIT=$(GIT_COMMIT)
+	@echo GIT_BRANCH2=$(GIT_BRANCH2)
+	docker build -f Dockerfile.packagebuilder --build-arg GIT_COMMIT=$(GIT_COMMIT) -t cloud-gate-packagebuilder .
+	docker run cloud-gate-packagebuilder cat cloud-gate_${VERSION}-2_amd64.deb > cloud-gate_${VERSION}-2_amd64.deb
+	docker run cloud-gate-packagebuilder cat /go/bin/cloud-gate > cloud-gate.linux
```

Comment:
Greate contribution. Just one more requeste before landing:
Plrease extract the build rpm also. It would be in /root/rpmbuild/RPMS/x86_64/

Otherwise great work,

Reply:
The changes are addressed on #16. Closing this one.